dts/dts-e2e.robot: PC Engines always have had the Dasharo with SeaBIOS

DaniilKl opened the pull request:
PC Engines always have had the Dasharo with SeaBIOS so we should do only an update here.

Commit:
dts/dts-e2e.robot: PC Engines always have had the Dasharo with SeaBIOS,
so we should do only an update here.

Signed-off-by: Daniil Klimuk <daniil.klimuk@3mdeb.com>

Commit:
dts: add new transtition workflow, adjust PCE E2E tests

Related PR: https://github.com/Dasharo/dts-scripts/pull/87

Signed-off-by: Daniil Klimuk <daniil.klimuk@3mdeb.com>

Commit:
dts: e2e: add TEST_EFI_PRESENT="false"

Related PR: https://github.com/Dasharo/dts-scripts/pull/87

Signed-off-by: Daniil Klimuk <daniil.klimuk@3mdeb.com>

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
     Execute Command In Terminal
     ...    export DTS_TESTING="true" TEST_SYSTEM_VENDOR="PC Engines" TEST_SYSTEM_MODEL="APU2"
-    Execute Command In Terminal    export TEST_BIOS_VERSION="v4.19.0.1" TEST_BOARD_MODEL="APU2"
+    Execute Command In Terminal    export TEST_BIOS_VERSION="coreboot 24.04.00.05" TEST_BOARD_MODEL="APU2"
```
```suggestion
    Execute Command In Terminal    export TEST_BIOS_VERSION="coreboot v24.04.00.05" TEST_BOARD_MODEL="APU2"
```

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
     Execute Command In Terminal
     ...    export DTS_TESTING="true" TEST_SYSTEM_VENDOR="PC Engines" TEST_SYSTEM_MODEL="APU2"
-    Execute Command In Terminal    export TEST_BIOS_VERSION="v4.19.0.1" TEST_BOARD_MODEL="APU2"
+    Execute Command In Terminal    export TEST_BIOS_VERSION="coreboot 24.04.00.05" TEST_BOARD_MODEL="APU2"
```
```suggestion
    Execute Command In Terminal    export TEST_BIOS_VERSION="coreboot v24.04.00.05" TEST_BOARD_MODEL="APU2"
```

DaniilKl commented:
@m-iwanicki, regarding the OptiPlex: https://github.com/Dasharo/open-source-firmware-validation/compare/9dd33eb465b872fa8cbdca7a113b1d3723bbb248..80eabd8cb39e7b2c57307ba0fbf6ed40064123d2

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
     ...    export DTS_TESTING="true" TEST_SYSTEM_VENDOR="PC Engines" TEST_SYSTEM_MODEL="APU2"
-    Execute Command In Terminal    export TEST_BIOS_VERSION="v4.19.0.1" TEST_BOARD_MODEL="APU2"
+    Execute Command In Terminal
+    ...    export TEST_BIOS_VERSION="coreboot 24.04.00.01" TEST_BOARD_MODEL="APU2"
```
```suggestion
    ...    export TEST_BIOS_VERSION="coreboot v24.04.00.01" TEST_BOARD_MODEL="APU2"
```

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
     ...    export DTS_TESTING="true" TEST_SYSTEM_VENDOR="PC Engines" TEST_SYSTEM_MODEL="APU2"
-    Execute Command In Terminal    export TEST_BIOS_VERSION="v4.19.0.1" TEST_BOARD_MODEL="APU2"
+    Execute Command In Terminal
+    ...    export TEST_BIOS_VERSION="coreboot 24.04.00.01" TEST_BOARD_MODEL="APU2"
```
```suggestion
    ...    export TEST_BIOS_VERSION="coreboot v24.04.00.01" TEST_BOARD_MODEL="APU2"
```

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
+    Wait For Checkpoint And Write    ${DTS_CHECKPOINT}    ${DTS_TRANSITION_OPT}
 
-    # 5) User should not have access to Heads update without proper credentials:
+    # 4) User should not have access to Heads update without proper credentials:
```
```suggestion
    # 4) User should not have access to UEFI update without proper credentials:
```

m-iwanicki commented on dts/dts-e2e.robot:
```diff
@@ -791,96 +791,86 @@ E2E004.008 Dell Optiplex 9010 DPP update (Coreboot + UEFI -> Coreboot + UEFI) -
-    Set DUT Response Timeout    30s
-
-    # 5) User should not have access to Heads update without proper credentials:
+    # 4) User should not have access to Heads update without proper credentials:
```
```suggestion
    # 4) User should not have access to UEFI update without proper credentials:
```

DaniilKl committed:
dts: e2e: fix typos after review

Related PR: https://github.com/Dasharo/open-source-firmware-validation/pull/878

Co-authored-by: Michał Iwanicki <michal.iwanicki@3mdeb.com>

DaniilKl commented:
@m-iwanicki, the results:

![image](https://github.com/user-attachments/assets/969da583-7e87-4315-9f36-4bd9942d89e8)

Both fails for the PCE are beacuse of this PR not merged https://github.com/Dasharo/dts-configs/pull/2, and there is no update:

![image](https://github.com/user-attachments/assets/eb4803e4-7381-4290-bc39-0c4c4a7c4970)
